projects/npm-tools/packages/npm-scripts/src/utils/bundlerNamespace.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+function addNamespace(packageName, namespacingPackageName) {
+	const namespace = makeNamespace(namespacingPackageName);
+
+	if (packageName.indexOf('@') === 0) {
```

Comment:
:thinking: I changed that from `startsWith` because it threw an error (I always use `startsWith` because it also exists in Java, so I never recall there's `indexOf` :-) ). 

I assumed it had to do with node 10 (which is what we use in portal) and changed it without further testing. But it is true that it works in node 10 too, so it sure failed for a different reason :shrug:

Reply:
It definitely works with v10:

```
❯ node --version
v10.15.1
❯ node -e 'console.log("foo".startsWith("f"))'
true
```

and [according to MDN, works since v4](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith).

Reply:
Yes, yes. I did the test here too.